### PR TITLE
Uplift structure of files, commands, and message rules

### PR DESCRIPTION
**Pull request by 20BBrown14**
Completely alter, clean up, and organize the file structure so it's easy to maintain and understand.

### CODE REVIEW

File: commands/emojify.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  await send_message(client, message, content)
+
+def emojify(string):
+  emojiMessage = ''
+  for c in string:
+    if not c.isalpha():
+      emojiMessage += c
+      continue
+    emojiMessage += ':regional_indicator_' + c + ':'
+  return emojiMessage
```
Comment (fruech101) [inline commands/emojify.py:31]: can we add numbers?

File: commands/google.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# GOOGLE COMMAND NOT IN USE
```
Comment (fruech101) [inline commands/google.py:1]: y tho

Comment (20BBrown14) [reply commands/google.py:1]: Because I didn't want to make it work. it has weird bugs that I'll need to work out in the future.

Comment (fruech101) [reply commands/google.py:1]: +1

**Pull request merged**